app/components/map-measurement-tools.js

```diff
@@ -56,11 +138,12 @@ export default class MapMeasurementToolsComponent extends Component {
     const drawMode = type === 'line' ? 'draw_line_string' : 'draw_polygon';
     const { mainMap } = this;
     if (mainMap.get('drawMode')) {
-      draw.deleteAll();
+      // the following have been commented out, as we no longer want to clear all previous drawings unless the user hits the x button
```

Comment:
No need to commit commented out code. If we don't need it, delete it. If we have to bring it back, that's what git is for.